processor/otel/metrics.go

```diff
@@ -316,6 +414,23 @@ func (ms metricsets) upsert(timestamp time.Time, name string, attributes pdata.A
 			return true
 		})
 		ms.upsertOne(timestamp, elasticapmName, elasticapmAttributes, sample)
+	case "system.memory.usage":
+		// Translation of Otel memory metrics
+		// system.memory.usage (state=free) -> system.memory.actual.free
+		// system.memory.usage (state=used) -> system.memory.actual.used.bytes
+		var elasticapmName string
+		attributes.Range(func(k string, v pdata.AttributeValue) bool {
+			if k == "state" {
+				switch v.StringVal() {
+				case "used":
+					elasticapmName = "system.memory.actual.used.bytes"
```

Comment:
Do we use this? Looking at the Go agent, it appears the only system memory metrics produced are `system.memory.total` and `system.memory.actual.free`.

Reply:
You are right, `system.memory.actual.used.bytes` is not used by the APM UI. I will remove it from the set of translated metrics. The final list of exported APM metrics will then be : 
```
jvm.memory.{area}.{type}
jvm.gc.time
jvm.gc.count
```
```
system.memory.actual.free
system.memory.total
```
```
system.cpu.total.norm.pct
```

Reply:
Removed in 39a0bdc